sass/themes/schemas/components/dark/_navdrawer.scss

```diff
@@ -53,9 +53,8 @@ $dark-fluent-navdrawer: extend(
         ),
         item-active-background: (
             color: (
-                'primary',
+                'gray',
                 300,
-                0.12,
```

Comment:
it still has 0.12 opacity

Reply:
item active background is 300 in the UI kit.

<img width="856" alt="Screenshot 2024-04-23 at 20 41 13" src="https://github.com/IgniteUI/igniteui-theming/assets/21260963/89713c41-5cfe-493a-9da0-f7c5a435dd63">

Reply:
No, it's not. Ask somebody from the design team if you're not sure.

<img width="907" alt="Screenshot 2024-04-24 at 8 14 38" src="https://github.com/IgniteUI/igniteui-theming/assets/45598235/84426452-b64a-4f38-9acd-66552bde7507">

Reply:
My bad, I didn't inspect the correct element.